# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libisofs/fix-sgi-volhdr.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libisofs-1.5.6/libisofs/system_area.c.vanilla	2025-05-05 19:25:18.543333323 +0100
+++ libisofs-1.5.6/libisofs/system_area.c	2025-05-05 19:25:41.212309659 +0100
@@ -568,9 +568,9 @@
     /* 416 - 419 |          0 | Partition is volume header */
     iso_msb(buf + 416, 0, 4);
 
-    /* 432 - 435 |  part_blks | Number of 512 byte blocks in partition */
+    /* 432 - 433 |  part_blks | Number of 512 byte blocks in partition */
     iso_msb(buf + 432, blocks, 4);
-    iso_msb(buf + 444, 6, 4);
+    iso_msb(buf + 440, 6, 4);
 
     /* 504 - 507 |   head_chk | Volume header checksum  
                                 The two's complement of bytes 0 to 503 read
